src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -198,15 +177,6 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
     [joinMessage, onAcceptJoinMessage, showJoinMessage, showLockedMessage]
   );
 
-  const emptyTables = useMemo(
-    () =>
-      tables.filter((table) => !usersSeatedAtTables[table.reference].length),
-    [tables, usersSeatedAtTables]
-  );
-
-  const canStartTable =
-    emptyTables.length <= ALLOWED_EMPTY_TABLES_NUMBER && !isSeatedAtTable;
```

Comment:
is the logic changed intentionally? As far as I see, the conditions of adding tables changed.

Reply:
Actually, I got an idea on how to do it even better

Reply:
@sunny-viktoryia so, I reworked the bug fix a bit to preserve the previous logic in cc5b02bd685dd6ea627055a4eb3a2b8fcebdb3b4